Libraries/Opc.Ua.Client.ComplexTypes/TypeBuilder/ComplexTypeFieldBuilder.cs

```diff
@@ -125,10 +125,14 @@ public Type CreateType()
             m_structureBuilder = null;
             return complexType;
         }
+
+        /// <summary>
+        /// Defines and creates types
+        /// </summary>
+        public TypeBuilder StructureTypeBuilder { get => this.m_structureBuilder; }
```

Comment:
public Type GetStructureType(int valueRank) { get => valueRank == ValueRanks.Scalar ? this.m_structureBuilder : this.m_structureBuilder.MakeArrayType(valueRank);